curl.sh

```diff
@@ -1,10 +1,39 @@
 package: curl
-version: "1.0"
-system_requirement_missing: |
-  curl and its development package are missing from your system.
-   * RHEL-compatible systems: you will probably need "curl" and "curl-devel" packages.
-   * Ubuntu-compatible systems: you will probably need "curl" and "libcurl4-openssl-dev" (or "libcurl4-gnutls-dev").
-system_requirement: ".*"
-system_requirement_check: |
-  curl --version > /dev/null; if test $? = 127; then exit 1; else printf "#include <curl/curl.h>\nint main() {}\n" | cc -xc -lcurl - -o /dev/null || exit 2; fi; exit 0
+version: "7.70.0"
+tag: curl-7_70_0
+source: https://github.com/curl/curl.git
```

Comment:
@sawenzel @Barthelemy is this fine for the CCDB API?

Reply:
Here I have to comment, that the whole libjalien development was started to let CCDB client use curl + JAliEn SSL features (tokens, CAs). I've tested this case and I am ready to make a PR to AliceO2 as soon as these recipes are good to be merged.

One more point here is that curl executable is used in scripts like freetype.sh:16 - it also works well (in my local and jalien-root CI tests), though there were some issues during development, now solved.

7.70.0 is the latest tagged curl version at the moment.